### PR TITLE
refactor: factory partition size changed to 100MB

### PR DESCRIPTION
**Pull request by JoergZeidler**
Factory partition should be big enough to persists a custom docker image archive file.

### CODE REVIEW

File: conf/machine/genericx86-64.extra.conf [hunk 1]
```diff
@@ -37,7 +37,7 @@ WICVARS:append = " OMNECT_GRUBENV_FILE"
 OMNECT_GRUBENV_FILE = "${LAYERDIR_omnect}/files/grubenv"
 
 # adu compatibility id
-OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "0"
+OMNECT_ADU_DEVICEPROPERTIES_COMPATIBILITY_ID = "1"
```
Comment (mlilien) [inline conf/machine/genericx86-64.extra.conf:40]: not sure this is really a problem for adu ... the adu update doesn't touch the factory partition and other partition sizes didn't change. i assume the adu update works seamlessly.

File: README.md [hunk 2]
```diff
@@ -83,9 +83,10 @@ Device         Boot   Start      End  Sectors  Size Id Type
 - `mmcblkXp7` is the writable `etc` overlay partition (ext4 filesystem mounted as overlayfs on `/etc`)
 - `mmcblkXp8` is the writable `data` partition with ext4 filesystem
 
-**Note1**: The size of `data` depends on your sdcard/emmc/nvme size. Per default it has a size of 512M and is resized on the first boot to the max available size.<br>
-**Note2**: The size of `rootA` and `rootB` depends on your board and image variant (development or release).<br>
-**Note3 (only for images where u-boot is used as OS bootloader)**: There is a reserved area between the boot partition and the rootA partition used for two redundant u-boot environment banks. For this purpose, the following configuration variables are used:
+**Note1**: The partition layout is just an example and may differ in a series product.<br>
```
Comment (mlilien) [inline README.md:86]: -> "The partition layout is an example, sizes may differ depending on your board and image variant (development or release)."
\+ delete note 3?

File: conf/machine/genericx86-64.extra.conf (hunk 1 above)
Comment (JoergZeidler) [reply conf/machine/genericx86-64.extra.conf:40]: good point; think you are right, this is not a breaking change... will test with adu

File: README.md (hunk 2 above)
Comment (JoergZeidler) [reply README.md]: fine for me, done

File: conf/machine/genericx86-64.extra.conf (hunk 1 above)
Comment (JoergZeidler) [reply conf/machine/genericx86-64.extra.conf]: test with adu was successfully

**Review by mlilien — APPROVED** [review]

**Pull request merged**